ios/RNLinksdk.m

```diff
@@ -211,6 +211,11 @@ - (void)stopObserving {
         self.linkViewController = [[PLKPlaidLinkViewController alloc] initWithConfiguration:linkConfiguration
                                                                               delegate:self.linkViewDelegate];
     }
+    
+    // self.linkViewController.modalPresentationStyle = UIModalPresentationStyle.UIModalPresentationFullScreen;
+    // UIModalPresentationFullScreen = 0. I don't know what to import to get UIModalPresentationStyle enum
+    self.linkViewController.modalPresentationStyle = 0;
```

Comment:
The Objective-C enum name is `UIModalPresentationFullScreen` — mind replacing that? thanks!

I will check on whether we can change the default to full screen — we currently defer to the iOS version default which changed in iOS 13.

Reply:
fixed